src/commands/image/deployImageApi/deployImage.ts

```diff
@@ -33,19 +31,12 @@ export async function deployImage(context: IActionContext & Partial<ContainerReg
 
     wizardContext.telemetry.properties.revisionMode = containerApp.revisionsMode;
 
-    const promptSteps: AzureWizardPromptStep<DeployImageApiContext>[] = [
-        new ImageSourceListStep(),
-    ];
-
-    // If more than the image tag changed, prompt for ingress again
-    if (getImageNameWithoutTag(wizardContext.containerApp?.template?.containers?.[0].image ?? '') !== getImageNameWithoutTag(wizardContext.image ?? '')) {
-        promptSteps.push(new IngressPromptStep());
-    }
-    promptSteps.push(new ContainerAppOverwriteConfirmStep());
-
     const wizard: AzureWizard<DeployImageApiContext> = new AzureWizard(wizardContext, {
         title: localize('deploy', 'Deploy image to container app "{0}"', containerApp.name),
-        promptSteps,
+        promptSteps: [
```

Comment:
Can simplify the steps added here because `ContainerRegistryImageConfigureStep` would now determine whether an `IngressPromptStep` needs to be added